src/py42/clients/securitydata.py

```diff
@@ -279,18 +271,15 @@ def _get_device_file_version(self, device_guid, md5_hash, sha256_hash, path):
         response = self._preservation_data_service.get_file_version_list(
             device_guid, md5_hash, sha256_hash, path
         )
-        versions = response[u"versions"]
+        versions = response.data.get(u"securityEventVersionsMatchingChecksum") \
```

Comment:
This is the main logic change. The response no longer has `versions`, and instead returns an obj that has these three props. We want to use whichever is the first of these to have any data in them.

A customer cannot have both xfc and pds so `preservationVersions` will always be empty if either of the other two are not, or vice-versa.